docker-services.yml

```diff
@@ -105,3 +105,13 @@ services:
     environment:
       - "OPENSEARCH_HOSTS=http://search:9200"
       - "DISABLE_SECURITY_DASHBOARDS_PLUGIN=true"
+  iipserver:
+    image: registry.cern.ch/docker.io/iipsrv/iipsrv:latest
+    ports: 
+      - "8080:80"
+    expose:
+      - "8080"
+    environment:
+      - "CORS=*"
+    volumes:
+      - ${INSTANCE_PATH}/iiif/images:/images # INSTANCE_PATH must be defined in the environment (or .env file)
```

Comment:
This is a big assumption, currently, the instance path is stored in `./.invenio.private` (not a `.env` file). Any suggestions on how to enforce this?